app/javascript/styles/partials/_extends.scss

```diff
@@ -35,7 +35,7 @@
     position: absolute;
     top: 0;
     left: 0;
-    z-index: -1;
+    z-index: -2;
```

Comment:
Not a comment on the z-index, but just a general codebase question: where does `_partial-bg` get applied? I tried searching via text match and got results in Community Profiles, categories, and datasets but am stuck from there.

Reply:
It's the gray background element. I'm not quite certain where it is inserted, but having z-`index: -1` was screwing with the animation I added. I would need to dig deeper to find.